testnet/0238fae4cb17f1cd903e678bb723e85c941e0cd77c9b9c7c0ee4b71bfd47847ecb.json

```diff
@@ -1,10 +1,10 @@
 {
   "id": 121,
-  "name": "",
+  "name": "FTP",
   "secp": "0238fae4cb17f1cd903e678bb723e85c941e0cd77c9b9c7c0ee4b71bfd47847ecb",
   "bls": "a2d107e3506f50af5d6248a37c2d11482ff37005da3b8c1018077f561b81b16cfa6a32c11cbf6c3fe5eaf8542454c077",
-  "website": "",
-  "description": "",
-  "logo": "",
-  "x": ""
+  "website": "https://ftptech.xyz",
+  "description": "FTP Tech is a trusted, infrastructure provider and active validator across multiple modular and app-specific chains with a focus on high performance, reliability and long-term alignment with the communities we support",
+  "logo": "https://res.cloudinary.com/def6oznlo/image/upload/v1752838833/IMG_6783_2_kqcuoz.jpg",
```

Comment:
Note: the logo image is quite large, I would suggest to submit a smaller one.

No a blocker though, let's merge.